Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -519,6 +519,10 @@ void HTMLInputElement::updateType()
     removeFromRadioButtonGroup();
     resignStrongPasswordAppearance();
 
+    bool oldTypeSupportsReadOnly = m_inputType->supportsReadOnly();
+    bool newTypeSupportsReadOnly = newType->supportsReadOnly();
```

Comment:
Nit: Rename to `didSupportReadOnly` / `willSupportReadOnly` matching the convention below.